### DISCUSSION TITLE
REQUEST: aarch64 build, please.

### DISCUSSION THREAD

**Discussion by 38github** [ZL-Audio/ZLEqualizer#67] [Ideas]
Would you be willing to provide also an aarch64 build, please?

**Comment by zsliu98** [comment]
I would assume that you are talking about Linux aarch64. Unfortunately, I cannot provide aarch64 build as neither I nor Github has a Linux aarch64 runner.
You may find some relevant resources at https://build.opensuse.org/package/show/multimedia:proaudio/ZLEqualizer, which is built by someone else.

**Reply by 38github** [reply]
Thank you so much! That solved it for me!

**Reply by zsliu98** [reply]
@38github GitHub has just provided Linux ARM64 runners. Therefore, I will release the ARM64 build (as long as GitHub still supports that). You may try the latest release `ZL.Equalizer-0.5.0-Linux-ARM64.zip`. I haven't tested it though. Hopefully it will work.

**Reply by 38github** [reply]
Thank you! It is really appreciated!
